third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -16,14 +16,15 @@ typedef std::function<SmallVector<Value>(Location, ConversionPatternRewriter &,
     ConverterT;
 
 namespace {
-/* ----- FP8E5M2 ------ */
+// ROCM utility functions for data type conversion
 /* ----- FP8E5M2 ------ */
 // This data-type is the standard FP8E5M2 format
-#ifdef USE_ROCM
-
-static SmallVector<Value> Fp16_to_Fp8E5M2(Location loc,
-                                          ConversionPatternRewriter &rewriter,
-                                          const SmallVector<Value> &v) {
+// This function is not fully functional for the denorm values
+// And currently it is not used as we don't have HW native support on
+// MI300. We will rewrite it if upstream adds the support for A100
+static SmallVector<Value>
+Fp16_to_Fp8E5M2_RTNE(Location loc, ConversionPatternRewriter &rewriter,
```

Comment:
These function names are not following the convention. Maybe name it something like `fp16ToFP8E5M2withRTNE`.

Reply:
We use the same function names with the NV side. Perhaps we can keep it like this.

Reply:
Okay that's fine then for now.